### ISSUE TITLE
Noisy rules_android autoload warning

### ISSUE THREAD

**Issue by keith** [bazelbuild/bazel#24597]
### Description of the bug:

In an empty bazel project with the latest 8.x rc you see this warning:

```
WARNING: Couldn't auto load rules or symbols, because no dependency on module/repository 'rules_android' found. This will result in a failure if there's a reference to those rules or symbols.
```

If your project doesn't support android, there's nothing you can do here (besides unnecessarily add this dep)

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

```sh
mkdir /tmp/repro
cd /tmp/repro
touch MODULE.bazel BUILD.bazel
USE_BAZEL_VERSION=last_rc bazelisk build ...
```

### What is the output of `bazel info release`?

8.0.0rc7

**Comment by benjaminp** [comment]
https://github.com/bazelbuild/bazel/issues/23929#issuecomment-2408459588

**Comment by keith** [comment]
nice, thanks for the workaround, i still don't think we want to force users to do that tho

**Comment by ahumesky** [comment]
This should be fixed with the 0.6.0 release of rules_android https://github.com/bazelbuild/rules_android/releases/tag/v0.6.0

**Comment by iancha1992** [comment]
A fix for this issue has been included in [Bazel 8.0.1 RC1](https://github.com/bazelbuild/bazel/releases/tag/8.0.1rc1). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting USE_BAZEL_VERSION=8.0.1rc1. Thanks!
